.infrastructure/deployment.yml

```diff
@@ -29,7 +29,15 @@ spec:
             cpu: "150m"
         env:
         - name: PYTHONUNBUFFERED
-          value: "1"
+          valueFrom:
+            confifMapKeyRef:
```

Comment:
There is a typo here: `confifMapKeyRef` should be `configMapKeyRef` to correctly reference the ConfigMap.

Reply:
done